repository/project.go

```diff
@@ -86,7 +86,12 @@ func (r *ProjectRepository) readProject(ctx context.Context, q querier, query st
 		return svcmodel.Project{}, err
 	}
 
-	return r.toSvcProject(p)
+	repo, err := r.toSvcGithubRepo(p)
+	if err != nil {
+		return svcmodel.Project{}, fmt.Errorf("failed to convert github repo: %w", err)
+	}
+
+	return model.ToSvcProject(p, repo), nil
```

Comment:
When fetching project/projects from repository. I also had two functions:

```
r.toSvcProject
r.toSvcProjects
```

These functions were removed (to follow your suggestion for release repo). There is only one helper method ` r.toSvcGithubRepo`.